MetaMask/metamask-mobile
# test: reorg signatures tests to maximize efficiency

## **Description**
In this PR we are splitting up the Signatures test into different files, in the same way we have on Extension:
- eth-sign
- personal-sign
- typed-sign

This helps us run test faster since they can run in parallel: ~ **9 minutes faster in Android**

## **Manual testing steps**
- Run tests locally
- Check Bitrise run here https://app.bitrise.io/app/be69d4368ee7e86d/pipelines/c6617bd3-452f-4147-ab73-8c3e201bb1ba

## **Screenshots/Recordings**

### **Before**
Running all tests (Smoke + Confirmations)

![image](https://github.com/MetaMask/metamask-mobile/assets/54408225/12733fdc-0827-4e14-8466-c997982bd4e8)


### **After**
Running all tests (Smoke + Confirmations)

![image](https://github.com/MetaMask/metamask-mobile/assets/54408225/13de19f4-9725-424b-93bb-9a6bd1671210)


## **Related issues**

_Fixes https://github.com/MetaMask/MetaMask-planning/issues/1394

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
